mycroft/skills/core.py

```diff
@@ -527,6 +527,73 @@ def shutdown(self):
             logger.error("Failed to stop skill: {}".format(self.name),
                          exc_info=True)
 
+    def _schedule_event(self, handler, datetime, data={}, name=None,
+                        repeat=None):
+        """
+            Underlying method for schedle_event and schedule_repeating_event.
+            Takes scheduling information and sends it of on the message bus
+        """
+        if not name:
+            name = self.name + handler.__name__
+        name = str(self.skill_id) + ':' + name
+        self.add_event(name, handler, False)
+        event_data = {}
+        event_data['time'] = time.mktime(datetime.timetuple())
+        event_data['event'] = name
+        event_data['repeat'] = repeat
+        event_data['data'] = data
+        self.emitter.emit(Message('mycroft.scheduler.schedule_event',
+                                  data=event_data))
+
+    def schedule_event(self, handler, datetime, data={}, name=None):
+        """
+            schedule a single event
+
+            Args:
+                handler:    method to be called
+                datetime:   time for calling the handler
```

Comment:
Lets write/doc it like this:
      when (datetime):  when the handler should be called
      data (dict, optional):  data to send when the handler is called
      name (str, optional):  friendly name for the event

So rename "datetime" to "when".  Make sense?

Reply:
Nemas problemas, I'll copy this to the wiki as an example for future reference